backends/gdx-backends-gwt/src/com/badlogic/gwtref/client/ReflectionCache.java

```diff
@@ -44,10 +44,10 @@ private static String convert (String className) {
 		if (className.startsWith("[")) {
 			int dimensions = 0;
 			char c = className.charAt(0);
-			String suffix = "";
+			StringBuilder suffix = new StringBuilder();

```

Comment:
There is still some concatenation after this (right below the GitHub's code fold) which could be optimized. Without it, this optimization will likely only slightly hurt, since most of the input `className`s used here are probably not arrays (especially not multi-dimension arrays).

Reply:
[fix](https://github.com/libgdx/libgdx/pull/5670/commits/36b03a102e920f5d244daddcc40e93589e6e57b1)